recipes/r-picante/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = '1.7' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+{% set cran_mirror = 'https://cran.r-project.org' %}
+
+package:
+  name: r-picante
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: picante_{{ version }}.tar.gz
+  url:
+    - {{ cran_mirror }}/src/contrib/picante_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/picante/picante_{{ version }}.tar.gz
+  sha256: 75e4d73080db67e776562a1d58685438461cbde39af46900c7838da56aef0a62
+
+build:
+  number: 0
+  skip: true  # [win32]
```

Comment:
You might want to consider just skipping this set on win altogether and do windows compatibility in the generated feedstock